src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -95,7 +95,15 @@ if [ "$(uname -s)" == "Darwin" ]; then
 elif [ "$(uname -s)" == "Linux" ]; then
   # On Linux, we'll enable core file generation unconditionally, and if a dump
   # is generated, we will print some useful information from it and delete the
-  # dump immediately. 
+  # dump immediately.
+
+  if [ -e /proc/self/coredump_filter ]; then
```

Comment:
@ellismg recommended putting the core dump logic here. I'm not too familiar with this codebase, but it seems like this is a good place to set up dump generation because it will be enabled only for the shell that this script runs in (which is pretty close to the minimum window we need it for).

Do you have another place in mind that would be more suitable?